_assets/ci/Jenkinsfile.tests-benchmark

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.24'
+
+pipeline {
+  agent { label 'benchmark' }
+
+  triggers {
+    // Nightly at 2am
+    cron 'H 2 * * *'
+  }
+
+  parameters {
+    string(
+      name: 'BRANCH',
+      defaultValue: 'develop',
+      description: 'Name of branch to build.'
+    )
+    booleanParam(
+      name: 'FUNCTIONAL_TESTS_REPORT_CODECOV',
+      defaultValue: true,
+      description: 'Should the job report test coverage to Codecov?'
+    )
+    choice(
+      name: 'FUNCTIONAL_TESTS_LOG_LEVEL',
+      choices: ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+      description: 'pytest log level'
+    )
+  }
+
+  options {
+    timestamps()
+    /* Prevent Jenkins jobs from running forever */
+    timeout(time: 50, unit: 'MINUTES')
+    disableConcurrentBuilds()
+    disableRestartFromStage()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: getAmountToKeep(),
+      daysToKeepStr: '-1',
+      artifactNumToKeepStr: getAmountToKeep(),
+    ))
+  }
+
+  environment {
+    PLATFORM = 'tests-benchmark'
+    PKG_URL = "${currentBuild.absoluteUrl}/consoleText"
```

Comment:
This doesn't really do anything if it's not a PR build that updated that GitHub comment.